test/src/test/java/hudson/slaves/CloudTest.java

```diff
@@ -85,9 +94,37 @@ public void cloudNameIsEncodedInGetUrl() {
         assertEquals("Cloud name is encoded in Cloud#getUrl", "cloud/..%2F..%2Fgibberish/", aCloud.getUrl());
     }
 
+    @Test
+    @Issue("JENKINS-71737")
+    public void changeCloudName() throws Exception {
+        ACloud aCloud = new ACloud("a", "0");
+        j.jenkins.clouds.add(aCloud);
+        JenkinsRule.WebClient webClient = j.createWebClient();
+        HtmlPage page = webClient.goTo(aCloud.getUrl() + "confirm-rename");
+        HtmlForm form = page.getFormByName("config");
+        HtmlTextInput input = form.getInputByName("newName");
+        input.setText("b");
+        j.submit(form);
+        ACloud actual = j.jenkins.clouds.get(ACloud.class);
+        assertEquals("b", actual.getDisplayName());
+    }
+
+    @Test
+    @Issue("JENKINS-71737")
+    public void changeCloudNameinConfig() throws Exception {
+        ACloud aCloud = new ACloud("a", "0");
+        j.jenkins.clouds.add(aCloud);
+        HtmlForm form = j.createWebClient().goTo(aCloud.getUrl() + "configure").getFormByName("config");
+        HtmlTextInput input = form.getInputByName("_.name");
+        input.setText("b");
+        Exception ex = assertThrows(FailingHttpStatusCodeException.class, () -> j.submit(form));
```

Comment:
Not sure how to pull up the error page that would say "Rename cloud using the "Rename" page"